src/facade/dragonfly_connection.cc

```diff
@@ -1400,11 +1421,11 @@ unsigned Connection::WeakRef::Thread() const {
 }
 
 Connection* Connection::WeakRef::Get() const {
-  DCHECK_EQ(ProactorBase::me()->GetPoolIndex(), int(thread_));
-  // The connection can only be deleted on this thread, so
-  // this pointer is valid until the next suspension.
-  // Note: keeping a shared_ptr doesn't prolong the lifetime because
-  // it doesn't manage the underlying connection. See definition of `self_`.
+  // DCHECK_EQ(ProactorBase::me()->GetPoolIndex(), int(thread_));
```

Comment:
@dranikpg somehow I still need to comment this line although I have been using `IsExpired()`